### PR TITLE
Pccs

### PR DESCRIPTION
**Pull request by kenplusplus**

### CODE REVIEW

File: attestation/pccs-k8s/chart/values.yaml [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+nodeSelector: {}
+
+tolerations: []
+
+affinity: {}
+
+# Configurations for DCAP PCCS
+PCCSConfig:
+  uri: https://api.trustedservices.intel.com/sgx/certification/v4/
+  apiKey: 829e0dc47bdb409e92a5757fc80422b6
```
Comment (ruomengh) [inline attestation/pccs-k8s/chart/values.yaml:59]: Will this trigger security scan violation?

Comment (kenplusplus) [reply attestation/pccs-k8s/chart/values.yaml:59]: Not sure, but for the default values.yaml, sound like we have to give a number, @dongx1x  is that right?

Comment (ruomengh) [reply attestation/pccs-k8s/chart/values.yaml:59]: It should be OK if this is a fake API key and token.

Comment (kenplusplus) [reply attestation/pccs-k8s/chart/values.yaml:59]: I will change all fake key to 0000

**Pull request merged**